<?xml version="1.0"?>
<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
    title="Valid Hook Name"
    >
    <standard>
    <![CDATA[
    Use lowercase letters in action and filter names. Separate words using underscores.
    ]]>
    </standard>
    <code_comparison>
        <code title="Valid: Lowercase hook name.">
        <![CDATA[
do_action( <em>'prefix_hook_name'</em>, $var );
        ]]>
        </code>
        <code title="Invalid: Mixed case hook name.">
        <![CDATA[
do_action( <em>'Prefix_Hook_NAME'</em>, $var );
        ]]>
        </code>
    </code_comparison>
    <code_comparison>
        <code title="Valid: Words separated by underscores.">
        <![CDATA[
apply_filters( <em>'prefix_hook_name'</em>, $var );
        ]]>
        </code>
        <code title="Invalid: Using non-underscore characters to separate words.">
        <![CDATA[
apply_filters( <em>'prefix\hook-name'</em>, $var );
        ]]>
        </code>
    </code_comparison>
</documentation>
